sentry-core/src/lib.rs

```diff
@@ -79,37 +79,31 @@
 //!
 //! Default features:
 //!
-//! * `with_client_implementation`: Turns on the real client implementation.
+//! * `client`: Turns on the real client implementation.
 //!
-//! Additional integrations:
+//! Additional features:
 //!
-//! * `with_debug_to_log`: When enabled sentry will debug log to a debug log at all times.
-//!
-//! Testing:
-//!
-//! * `with_test_support`: Enables the test support module.
+//! * `log`: When enabled sentry will debug log to `log` at all times.
+//! * `test`: Enables the test support module.
+
 #![warn(missing_docs)]
 
 #[macro_use]
 mod macros;
-
 mod api;
 mod breadcrumbs;
-mod hub;
-mod scope;
-
-pub mod integrations;
-
-#[cfg(feature = "with_client_implementation")]
+#[cfg(feature = "client")]
 mod client;
 mod clientoptions;
 mod constants;
 mod error;
+mod hub;
+mod integration;
 mod intodsn;
-mod transport;
-
-#[cfg(any(test, feature = "with_test_support"))]
+mod scope;
+#[cfg(any(test, feature = "test"))]
```

Comment:
Would you mind adding the blank line again and moving the `test` module into its own group? Otherwise, it's too easy to mess up the feature flags. Especially, since it's also a public module, it should go in its own group.